app/javascript/mastodon/features/list_timeline/index.js

```diff
@@ -187,6 +195,13 @@ class ListTimeline extends React.PureComponent {
             </button>
           </div>
 
+          <div className='setting-toggle'>
+            <Toggle id='lists.exclusive' defaultChecked={isExclusive} onChange={this.onExclusiveToggle} />
+            <label htmlFor='lists.exclusive' className='setting-toggle__label'>
+              <FormattedMessage id='lists.exclusive' defaultMessage='Hide these toots from home' />
```

Comment:
I love the use of "toots" here, but upstream is desperate to remove "toot", so this wouldn't be accepted.
```suggestion
              <FormattedMessage id='lists.exclusive' defaultMessage='Hide these posts from home' />
```

Reply:
Heh, didn't know that. Toots waa finally starting to grow on me 😜